src/Routes/README.md

```diff
@@ -1,3 +1,206 @@
 ## Routes
 
-Examples can be found in: `wp-content/plugins/core/src/Routes`
+Examples can be found in: `wp-content/plugins/core/src/Routes/`
+
+### Registering a route
+#### REST Route
+Create a new class in `wp-content/plugins/core/src/Routes/` that extends `Tribe\Libs\Routes\Abstract_REST_Route`
+
+    <?php declare( strict_types=1 );
+	/**
+	 * Sample REST route.
+	 *
+	 * @package Project
+	 */
+	
+	namespace Tribe\Project\Routes;
+
+	use Tribe\Libs\Routes\Abstract_REST_Route;
+
+	/**
+	 * Class for an example REST route.
+	 */
+	class Sample_REST_Route extends Abstract_REST_Route {
+		/**
+		 * Registers routes.
+		 *
+		 * @return void
```

Comment:
` * @return void` can go, no need for return types if the method itself has one.